udp-rsc-offload.md

```diff
@@ -42,7 +42,12 @@ URO can only be attempted on a batch of packets that meet **all** the following
 - The UDP checksums on pre-coalesced packets must be correct. This means checksum offload must be enabled and set the checksum OOB info.
 - TTL, ToS/ECN, Protocol, and DF bit must match on all packets (IPv4).
 - TC/ECN, FlowLabel, and HopLimit must match, and NextHeader must be UDP (IPv6).
-- The total length of the Single Coalesced Unit (SCU) must not exceed IP max length.
+- The total length of the Single Coalesced Unit (SCU) is allowed to exceed IP max length.
+- When the SCU length is larger than the max IP length:
+  - Set NB->DataLength to the coalesced size (up to 0xFFFFFFFF).
+  - Set IP total length field to 0.
+  - Set UDP length field to 0 when UDP length is greater than max UDP length.
+  - Set UDP checksum field to 0.
```

Comment:
Why are these conditional? IMO that raises complexity on both NICs and the stack, but offers no particular benefit. If we're going to support zeros in these fields for URO, why not make it the default? My preference is to say: URO indications _should_ set the IP length, UDP length, and UDP checksum fields to zero. Components handling URO indications _must_ ignore the IP length, UDP length, and UDP checksum fields.

Reply:
Yeah, you make a good point. I'll put your suggestion in a new PR.